<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultipleQustion</class>
 <widget class="QDialog" name="MultipleQustion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #5A5856;</string>
  </property>
  <widget class="QRadioButton" name="answer1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #FCA408;font-weight: 900;font-size: 17px;</string>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="answer2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #FCA408;font-weight: 900;font-size: 17px;</string>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="answer3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #FCA408;font-weight: 900;font-size: 17px;</string>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="answer4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #FCA408;font-weight: 900;font-size: 17px;</string>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(22, 23, 75);
color: rgb(0, 255, 0);</string>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="format">
    <string>%vs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="skipbot">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #36C3FF ; 
 font-weight: 1000; 
 font-size: 17px; 
 border: none; </string>
   </property>
   <property name="text">
    <string>skip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="verifybot">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: #36C3FF ; 
 font-weight: 1000; 
 font-size: 17px; 
 border: none; </string>
   </property>
   <property name="text">
    <string>ثبت پاسخ</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="quesText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #9D9DFA ;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;علی کسیت؟&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="category">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #29FF43;font-weight: 900;font-size: 17px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="cat_text">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: yellow;font-weight: 900;font-size: 17px;
</string>
   </property>
   <property name="text">
    <string>موضوع:</string>
   </property>
  </widget>
  <widget class="QLabel" name="type1_label">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>70</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #29FF43;font-weight: 900;font-size: 17px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="typetxt">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: yellow;font-weight: 900;font-size: 17px;</string>
   </property>
   <property name="text">
    <string>نوع:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
